docs/updates/2020-08-07-announce-book.md

```diff
@@ -0,0 +1,108 @@
+
+MyST Markdown is a *superset* of Jupyter Markdown (AKA, CommonMark), meaning that *any default markdown in a Jupyter Notebook is valid in Jupyter Book*. If you'd like extra features in markdown such as citations, figures, references, etc, then you may include extra MyST Markdown syntax in your content.
+
+For example, here's how you can [include a citation](https://jupyterbook.org/content/citations.html) in the new Jupyter Book:
+
+```{figure} images/citation-example.png
+
+A sample citation. Here we see how you can include citation syntax in-line with your markdown, and then insert a bibliography later on in your page.
+```
+
+And here's how you can include a figure:
+
+```{figure} images/figure-example.png
+
+A sample figure. Here we see how you can insert a figure with a caption into your book,
+and control some aspects of how the figure is displayed with options.
+```
+
+### A smarter build system
+
+While the old version of Jupyter Book used a combination of Python and Jekyll to build your book's HTML, the new Jupyter Book uses Python all the way through. This means that building the HTML for your book is as simple as:
+
+```
+jupyter-book build mybookname/
+```
+
+In addition, the new build system leverages [Jupyter Cache](https://jupyter-cache.readthedocs.io/) to execute notebook content *only if the code is updated*, and to insert the outputs from the cache at build time. This saves you time by avoiding the need to re-execute code that hasn't been changed.
+
+```{figure} images/build-process.gif
+
+An example build process. Here the `jupyter-book` command-line interface is used to convert a collection of content into an HTML book.
+```
+
+### More book output types
+
+By leveraging Sphinx, Jupyter Book will be able to support more complex outputs than *just* an HTML website. For example, we are currently prototyping **[PDF Outputs](https://jupyterbook.org/advanced/pdf.html)**, both via HTML as well as via Latex. This gives Jupyter Book more flexibility to generate the right book for your usecase.
+
+You can also run Jupyter Book on **[individual pages](https://jupyterbook.org/start/build.html#build-a-standalone-page)**. This means that you can write single-page content (like a scientific article) entirely in Markdown.
+
+### A new stack
+
+The biggest change under-the-hood is that Jupyter Book now uses [the Sphinx documentation engine](https://www.sphinx-doc.org/en/master/) instead of Jekyll for building books. By leveraging the Sphinx ecosystem, Jupyter Book can more effectively build on top of community tools, and can contribute components back to the broader community.
+
+Instead of being a single repository, the  old Jupyter Book repository has now been separated into several modular tools. **Each of these tools can be used on their own in your Sphinx documentation**, and they can be coordinated together via Jupyter Book:
```

Comment:
```suggestion
Instead of being a single repository, the old Jupyter Book repository has now been separated into several modular tools.
**Each of these tools can be used on their own in your Sphinx documentation**, and they can be coordinated together *via* Jupyter Book:
```